interface/modules/custom_modules/oe-module-claimrev-connect/templates/receiver.php

```diff
@@ -0,0 +1,43 @@
+ <?php
+ /**
+ *
+ * @package   OpenEMR
+ * @link      http://www.open-emr.org
+ *
+ * @author    Brad Sharp <brad.sharp@claimrev.com>
+ * @copyright Copyright (c) 2022 Brad Sharp <brad.sharp@claimrev.com>
+ * @license   https://github.com/openemr/openemr/blob/master/LICENSE GNU General Public License 3
+ */
+   
+if($receiver != null)
+{
+    $companyProviderCaption = "Company Name";
+    $companyProviderName = $receiver->lastOrganizationName;
+    if($receiver->entityIdentifierCodeQualifier == "1")
+    {
+        $companyProviderCaption = "Provider Name";
+        $companyProviderName = $receiver->firstName . " " . $receiver->middleName . " " .  $receiver->lastOrganizationName . " " . $receiver->suffix;
+    }
+?>
+    <div class="card">
+        <div class="card-body">
+            <h5 class="card-title">Receiver Information</h5>
+            <div class="row"> 
+                <div class="col">
+                    <?php echo xlt($companyProviderCaption); ?>                        
```

Comment:
What is an example of $companyProviderCaption (just trying to figure out if this needs to be translated) ?

Reply:
1. then use xl() in line 14 and 18 (this does translation without escaping); doing this will allow it to be captured in perl script we run to collect translation constants from codebase in the internationalization/translations pipeline.
2. then just use text() in line 27